src/schema/user.js

```diff
@@ -84,38 +168,76 @@ export const resolvers = {
       return userinfo?.attributes?.municipalityAgencyId;
     },
     async debt(parent, args, context, info) {
```

Comment:
Hvorfor er det egentlig ikke debts, men bare debt? Jeg ved godt at gæld er utælleligt, men har vi ikke kaldt det "debts" flere steder når vi kigger på alle "gælderne" som en bruger har?

Reply:
Ved ikke, den blev kaldt debt af Kasper for 2 år siden, har ikke ændret det her. 

Gæld kan returneres som både en liste eller et samlet gældsoverblik. Tror ikke der er taget ultimativ stilling til det endnu